daikon-logging/logging-event-layout/src/main/resources/mdc_ecs_mapping.properties

```
@@ -1,4 +1,4 @@
-accountId=cloud.account.id
+accountId=labels.customer_account_id
 executionId=labels.execution_id
 stackName=labels.stack_name
 userId=user.id
```

I would leverage labels for userId as well, so everything is aligned.
```suggestion
userId=labels.customer_user_id
```

---

You're right, it would be aligned, but using the dedicated field is always better, no ? I mean for analysis purpose.
Wdyt @sgandon, @rbailly-talend ?

---

It has been decided with Archs/SREs to leverage `organization.id` to store account/tenant id as it seems to be the more suitable field for that purpose. It seems now to be logical to use `user.id` in order to store user id.